index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css" rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="./style.css" />
+  <title> Portfolio</title>
+</head>
+
+<body>
+  <header id="header">
+    <div class="logo">
+      <a href="#">My Logo</a>
+    </div>
+    <nav class="navbar">
+      <ul class="nav-list">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#contact">Contact</a></li>
+      </ul>
+      <div class="menu">
+        <i class="bx bx-menu"></i>
+      </div>
+      <div class="close">
+        <i class="fa fa-times"></i>
+      </div>
+    </nav>
+    <div class="bar"></div>
+  </header>
+  <main class="main-container">
+    <section class="caption">
```

Comment:
- Kindly make sure you add the closing tag for the `<main></main>` `<section></section> `as I see you did open them but you didn't close[Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. :+1: